include/SSVOpenHexagon/Core/HexagonGame.hpp

```diff
@@ -250,11 +258,21 @@ class HexagonGame
         return game;
     }
 
+    sf::Vector2f getFieldPos() const noexcept
```

Comment:
It's good practice to mark functions whose return value should never be ignored as `[[nodiscard]]`, as accidentally ignoring the return value results in a compiler warning (rather than a silent run-time mistake).